translations/messages.de.yaml

```diff
@@ -12,6 +12,9 @@ ngsite.layout.sponsored: 'Gesponsert'
 ngsite.search.placeholder: 'Suche'
 ngsite.search.button: 'Suche'
 
+ngsite.tags.tag: 'Tag'
+ngsite.tags.related_tags: "Verwandte tags"
```

Comment:
Ha, now this is wrong. In German, this actually needs to be uppercase `T` :smile:

Reply:
haha, fixed